src/schemaToCSVTransforms.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash/fp'
+import { updateMany } from './futil'
+
+// Maps contexture schemas to tranforms for fast-csv
+export let schemaToCSVTransforms = (schema, {logger = _.noop, header = true, include } = {}) => {
+  let count = 0
+  let transformedHeaders = include && _.reduce((acc, k) => ({...acc, [k]: _.get(`${k}.label`, schema) || _.startCase(k) }), {}, include)
+
+  let headers = _.mapValues('label', schema)
+  return {
+    transformedHeaders,
+    transformHeaders: key => headers[key] || _.startCase(key),
+    // NOTE: for whatever reason you can't use a function declared
+    // by _.flow as transaform, just `transform: _.identity` works as expected
+    // but `transform: _.flow(_.identity)` does write a csv and doesn't throw any errors :(
+    // _.flow might be hitting the fast-csv callback api for transform but not really sure
+    transform: row => _.flow(
+      include ? _.pick(include) : _.identity,
```

Comment:
We may want to use F.when here

Reply:
I think we may choose to use either x=>x or _.identity not use both of them.

Reply:
We probably don't want to use F.when because this gets resolved before a function get's returned by `_.flow` (i.e. it's being used to compose the function and this makes it clearer it's not part of the flow but helping compose it).